AntWars/ConfigurationPanel.cs

```diff
@@ -163,6 +163,7 @@ partial class ConfigurationPanel : Form {
                 configLoadedOrNewCreatedGame();
                 checkPlayerKI();
                 loadGame(config);
+                config.GamePath = res;
```

Comment:
Kann auch in der Config gesetzt werden auf ``Config#loadConfig(String)`` so würde man sich logic im ConfigPanel sparen.